src/register.ts

```diff
@@ -110,7 +110,8 @@ const registerCommands = async <Ctx extends Context = Context, Req extends Reque
 
         // Get the props to patch and do the update
         const { command, discord, diff } = toPatch[i];
-        const cmdPatch = objectPatch(command, diff);
+        const commandRecord: Record<string, unknown> = { ...command };
```

Comment:
From `toPatch` above, we should know this is `Command<Ctx, Req, Sentry>`? Can typescript not automatically carry this through?

Reply:
For this, I'm not entirely sure. without shallow copying, it throws this error:

```
Argument of type 'Command<Ctx, Req, Sentry>' is not assignable to parameter of type 'Record<string, unknown>'.
  Index signature for type 'string' is missing in type 'Command<Ctx, Req, Sentry>'.ts(2345)
```